go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -205,6 +205,12 @@ func runOvnKube(ctx *cli.Context) error {
 		return nil
 	}
 
+	// start the prometheus server
+	metricsBindAddress := ctx.String("metrics-bind-address")
+	if metricsBindAddress != "" {
```

Comment:
@JacobTanenbaum since we added this to Config, you can just:

```
if config.Kubernetes.MetricsBindAddress != "" {
    ovncluster.StartMetricsServer(config.Kubernetes.MetricsBindAddress)
}
```

Reply:
Done